### ISSUE TITLE
SFAuthenticationSession Cookies not successed

### ISSUE THREAD

**Issue by JackSteven** [dvdhpkns/SFAuthenticationSession-example#1]
XCode: Version 9.0 (9A235)
Simulator: iPhone7 + iOS11 .0

hello，run your demo later, last step 6 is user cookie not set.

I don't know why?

**Comment by clintonb** [comment]
I see the same issue. Suspecting an issue with storing cookies for local hosts, I tried using localhost, instead of 0.0.0.0, and using ngrok. Neither worked. However, I the Instagram example at https://github.com/dvdhpkns/SFAuthenticationSession-Instagram-Oauth works without issue.

**Comment by dvdhpkns** [comment]
Yes, I see the same issue, I noted it in the README. I sent this repo to apple in a radar and they marked it as closed despite it not being resolved @JackSteven @clintonb.

**Comment by JackSteven** [comment]
Lately, I'm running to my iPhone7 (not Simulator), successed. Thanks. @clintonb @dvdhpkns .

**Comment by dvdhpkns** [comment]
Really @JackSteven? I tried running on my iPhone 7 today and was not successful. I went to Safari and set the cookie using my computer's IP address and then I pointed the app at the same IP and the cookie was not set. What iOS version are you on?

**Comment by JackSteven** [comment]
@dvdhpkns Sorry, I forgot to say, I tried running on my colleague‘s web development environment, and then successful.

**Comment by scpender** [comment]
Works great for me on a physical device. I had to tweak the app code's ViewController.swift file - Modified authURL to point to my server's IP (0.0.0.0 won't always work if not tethered)... @dvdhpkns  In the server setup instructions you skipped the "virtualenv env" line before source.
